lib/net/connections.fz

```diff
@@ -51,8 +51,20 @@ module:public connections(public active container.Set net.connection) : effect i
           fn.call
 
 
-  # NYI: UNDER DEVELOPMENT:
-  # in_thread_pool(c)
+  # this installs reader, writer and channel for the given connection
+  #
+  # NYI: DOCUMENTATION: see tests/sockets_thread_pool for now
+  #
+  public in_thread_pool(T type, LM type : mutate, lm LM, c net.connection, fn ()->T) concur.Future (outcome T) ! concur.thread_pool
```

Comment:
`LM.instate_self` does unfortunately not work. since `LM` is a type parameter and instate_self needs an instance. We somehow need an instance of the effect. Open to better ideas though.